apps/explorer/src/pages/transaction-result/Signatures.tsx

```diff
@@ -36,9 +37,12 @@ function SignaturePanel({
                     </Text>
                 </DescriptionItem>
                 <DescriptionItem title="Address">
-                    <Text variant="p1/medium" color="steel-darker">
-                        {signature.pubKey.toSuiAddress()}
-                    </Text>
+                    <AddressLink
+                        noTruncate
+                        address={normalizeSuiAddress(
```

Comment:
The sui address is already normalized in the latest changes in main.

Reply:
oh sorry for the bad advice then, didn't realize that!

Reply:
Made follow up PR to remove the normalizer here https://github.com/MystenLabs/sui/pull/9802